package.json

```diff
@@ -37,9 +37,9 @@
     "co-request": "^1.0.0",
     "eventemitter2": "^2.2.1",
     "five-bells-shared": "^23.0.0",
-    "ilp-core": "~13.0.0",
+    "ilp-core": "interledgerjs/ilp-core#mj-release-2.0",
```

Comment:
yeah, I think at merge time we should use the branch names in package.json, then:
* once things are merged, bump dependency versions on npm, e.g. ilp-core -> version 14
* then update the ilp-kit package.json
* then we can remove the branches from github